tests/build/test_launch_script.py

```diff
@@ -0,0 +1,146 @@
+# Copyright The FMS HF Tuning Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit Tests for accelerate_launch script.
+"""
+
+# Standard
+import base64
+import os
+import pickle
+import tempfile
+
+# Third Party
+import pytest
+
+# First Party
+from build.accelerate_launch import main
+from build.utils import INTERNAL_ERROR_EXIT_CODE, USER_ERROR_EXIT_CODE
+from tests.data import TWITTER_COMPLAINTS_DATA
+
+SCRIPT = "build/launch_training.py"
+MODEL_NAME = "Maykeye/TinyLLama-v0"
+BASE_PEFT_KWARGS = {
+    "model_name_or_path": MODEL_NAME,
+    "training_data_path": TWITTER_COMPLAINTS_DATA,
+    "num_train_epochs": 5,
+    "per_device_train_batch_size": 4,
+    "per_device_eval_batch_size": 4,
+    "gradient_accumulation_steps": 4,
+    "learning_rate": 0.00001,
+    "weight_decay": 0,
+    "warmup_ratio": 0.03,
+    "lr_scheduler_type": "cosine",
+    "logging_steps": 1,
+    "include_tokens_per_second": True,
+    "packing": False,
+    "response_template": "\n### Label:",
+    "dataset_text_field": "output",
+    "use_flash_attn": False,
+    "torch_dtype": "float32",
+    "max_seq_length": 4096,
+    "peft_method": "pt",
+    "prompt_tuning_init": "RANDOM",
+    "num_virtual_tokens": 8,
+    "prompt_tuning_init_text": "hello",
+    "tokenizer_name_or_path": MODEL_NAME,
+    "save_strategy": "epoch",
+    "output_dir": "tmp",
+}
+
+
+def test_successful_pt():
+    """Check if we can bootstrap and peft tune causallm models"""
+    os.environ["LAUNCH_TRAINING_SCRIPT"] = SCRIPT
+    os.environ["PYTHONPATH"] = "./:$PYTHONPATH"
```

Comment:
Is this what was needed in order for the build.utils to be imported correctly?

Reply:
yup.